replicatedhq/ship
# tag changes (Resolves #358)

What I Did
------------

Update tags to match specification in #358


How I Did it
------------

Modify goreleaser files


How to verify it
------------

`docker run replicated/ship:alpha` should now be the latest HEAD commit on master


Description for the Changelog
------------

Change how docker images are published. The `alpha` tag now references the current master branch. `latest` or a semver version should be used
to pull a stable release.


![](https://upload.wikimedia.org/wikipedia/commons/9/95/Red_Wharf_Bay_and_a_Shipwreck_2_-_geograph.org.uk_-_323541.jpg)
